harvest/api/alpaca.py

```diff
@@ -27,6 +27,12 @@ def __init__(
         paper_trader: bool = False,
     ):
         super().__init__(path)
+
+        if self.config is None:
+            raise Exception(
+                f"Account credentials not found! Expected file path: {path}"
+            )
+
```

Comment:
If the account is none then exit with an execption when the api is initalized rather than when self.config is accessed.